test/server/cards/02_SHD/units/MigsMayfeldTriggerman.spec.ts

```diff
@@ -0,0 +1,44 @@
+describe('Migs Mayfeld, Triggerman', function() {
+    integration(function(contextRef) {
+        describe('Migs Mayfeld\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['pillage', 'forced-surrender'],
+                        groundArena: [{ card: 'grogu#irresistible' }, { card: 'migs-mayfeld#triggerman' }],
```

Comment:
If there's no `exhausted`, `upgrades`, etc can we just do the string name instead of an object with a `card` property. just to keep the test setup as clean as possible

Reply:
Got it!